module/Education/src/Education/Controller/AdminController.php

```diff
@@ -12,6 +12,31 @@ public function indexAction()
     {
     }
 
+    public function addCourseAction()
+    {
+        $service = $this->getExamService();
+        $request = $this->getRequest();
+
+        if ($request->isPost()) {
+            // try uploading
+            if ($service->AddCourse($request->getPost())) {
```

Comment:
First character of a method or variable is always lowercase. Make it `addCourse`.